test/specs/wrapper/setProps.spec.js

```diff
@@ -244,6 +245,17 @@ describeWithShallowAndMount('setProps', mountingMethod => {
     expect(wrapper.vm.propA).to.equal('value')
   })
 
+  it('correctly sets props in async mode when component has immediate watchers', async () => {
+    const wrapper = mountingMethod(ComponentWithWatchImmediate, {
+      sync: false
```

Comment:
I might be missing something here, but I think `sync` is gone as per #1141?

Reply:
Correct.

Reply:
That's a typo. Thanks for the catch